content/_guides/creatingchannels.md

```diff
@@ -1,52 +1,196 @@
+registrations. It shows some additional information in `NickServ` and allows
+for convenient features such as `CLAIM` in `ChanServ`.
+
+If you already have a group registration set up with us, and you are listed as
+a Group Contact (GC) for the namespace you have chosen, it will be displayed
+to you in your `NickServ` info. Look for a line like:
+
+```irc
+NickServ: Group contact for projectname (#projectname; projectname/*)
+```
+
+`CLAIM` will allow you to acquire a channel _within your namespace_ even if
+that channel is already registered to someone else, without the need for staff
+intervention. It can also be used for initial registration even if it is not
+already taken.
+
+```irc
+09:23 -- ChanServ: ***** ChanServ Help *****
+09:23 -- ChanServ:  
+09:23 -- ChanServ: Help for CLAIM:
+09:23 -- ChanServ:  
+09:23 -- ChanServ: The CLAIM command allows you to take control of a channel
+09:23 -- ChanServ: belonging to a project you are authorized to represent.
+09:23 -- ChanServ:  
+09:23 -- ChanServ: Syntax: CLAIM <#channel>                            
+09:23 -- ChanServ:  
+09:23 -- ChanServ: Examples:
+09:23 -- ChanServ:     /msg ChanServ CLAIM #coolproject-dev            
+09:23 -- ChanServ:  
+09:23 -- ChanServ: ***** End of Help *****
+```
+
+Using `CLAIM` to grab a channel from someone else might be seen as hostile, or
+it could confuse people, so it is a good idea to at least give some warning.
+Try to ask for the cooperation of people who have already occupied or
+registered a channel before taking it from them.
+
+## Setting channel modes
+
+Channels differ in what `cmodes` they need. Refer to
+[the channel modes guide](channelmodes) for detailed information about them.
+
+The default modes set on a new channel are `+Cnst`.
+
+To set channel modes, you use `/mode <#channelname> [+|-]<mode>`.
+
+For example, during a spambot attack you might choose to only allow users who
+are [logged in](registration#logging_in) to join your channel. This is
+effective as most bots do not log in to accounts. To set the requirement, you
+would use the `r` mode.
+
+To toggle `r` on, so only identified users can join:
+
+```irc
+/mode <#channelname> +r
+```
+
+To toggle `r` off, so everyone else can join again:
+
+```irc
+/mode <#channelname> -r
+```
+
+## Setting up permissions
+
+Once you have your channel registered, and your users start dropping in, you
+might also need to do some [channel management](catalyst) if some of them
+outstay their welcome. If you enlist help for this, you will need to give your
+new Operators permissions.
+
+The permissions you can give are outlined in the output of
+`/msg ChanServ HELP FLAGS`, along with instructions about the use of
+`TEMPLATE` functionality.
+
+**Be very careful who you grant the `F` flag to. They will have full
+permissions on the channel and will be able to remove your own `F`.** The `f`
+flag will allow them to do everything **except** remove your `F`.
```

Comment:
`f` won't let them do *everything*, it will only let them set permissions they themselves already have

Reply:
i always get this wrong lol